src/Layer/LayerUpdateStrategy.js

```diff
@@ -11,8 +11,9 @@ export const STRATEGY_GROUP = 1;
 export const STRATEGY_PROGRESSIVE = 2;
 export const STRATEGY_DICHOTOMY = 3;
 
-function _minimizeNetworkTraffic(node, nodeLevel, currentLevel) {
-    if (node.pendingSubdivision) {
+function _minimizeNetworkTraffic(node, nodeLevel, currentLevel, source) {
+    // TO DO source.isVectorTileSource is a temp fix for pendingSubdivision
+    if (node.pendingSubdivision && !source.isVectorTileSource) {
```

Comment:
could you use `isVectorSource` ?

Reply:
all VectorTileSource are VectorSource but all VectorSource aren't VectorTileSource. I'm not sure we wnat to change the behaviour of all VectorSources here...